DCOS/templates/acs-engine-hybrid.json

```diff
@@ -79,6 +99,36 @@
           }
         ]
       }
-    }
+    },
+    "extensionProfiles": [
+      {
+        "name": "preprovision-agent-linux-public",
+        "version": "v1",
+        "extensionParameters": "parameters",
+        "rootURL": "http://13.68.86.214/preprovision/",
+        "script": "preprovision-agent-linux-public.sh"
+      },
+      {
+        "name": "preprovision-agent-linux-private",
+        "version": "v1",
+        "extensionParameters": "parameters",
+        "rootURL": "http://13.68.86.214/preprovision/",
+        "script": "preprovision-agent-linux-private.sh"
+      },
+      {
+        "name": "preprovision-master-linux",
+        "version": "v1",
+        "extensionParameters": "parameters",
+        "rootURL": "http://13.68.86.214/preprovision/",
+        "script": "preprovision-master-linux.sh"
+      },
+      {
+        "name": "preprovision-agent-windows",
+        "version": "v1",
+        "extensionParameters": "parameters",
+        "rootURL": "http://13.68.86.214/preprovision/",
+        "script": "preprovision-agent-windows.ps1"
+      }
+    ]
```

Comment:
I think we should change he rootURL to a different web server, one that can be relied to be online and unchanging.

Reply:
Hello John,

Would you leave the current web server online until I add these files to our CI web server available at [http://dcos-win.westus.cloudapp.azure.com](http://dcos-win.westus.cloudapp.azure.com) ?

This is considered to be online and unchanging. I'll be doing this by the end of this week.

Regards,
Ionut

Reply:
Will gladly leave the changes up on this web server.  The IP will very likely not change (since it's hosted in Azure).